src/test/java/com/google/sps/tool/GoogleTranslateTest.java

```diff
@@ -0,0 +1,29 @@
+package com.google.sps.tool;
+
+import org.junit.Test;
+import org.junit.Assert;
+import static org.junit.Assert.assertNotNull;
+import com.google.sps.tool.GoogleTranslate;
+
+public class GoogleTranslateTest {
+
+  @Test
+  public void translateHelloToSpanish() {
+    assertNotNull(GoogleTranslate.translateText("hello", "es"));
```

Comment:
i was thinking more along that lines of mocking out the call to the translate api itself. 

See, for example, https://www.baeldung.com/mockito-verify

Reply:
TODO(ngothomas): revisit with Zach later